jwst/assign_wcs/util.py

```diff
@@ -994,9 +994,9 @@ def compute_footprint_spectral(model):
     max_ra = np.nanmax(ra)
 
     # for the footprint we want the ra values to fall between 0 to 360
-    if(min_ra < 0):
+    if (min_ra < 0):
         min_ra = min_ra + 360.0
-    if(max_ra >= 360.0):
+    if (max_ra >= 360.0):
         max_ra = max_ra - 360.0
```

Comment:
```suggestion
    max_ra = min(max_ra, 360.0)
```

Reply:
I removed the brackets, but did not change the code since this is just a style PR.